master/pkg/schemas/defaults_test.go

```diff
@@ -0,0 +1,64 @@
+package schemas
```

Comment:
can you add a test for defaulting 1. an empty experiment config and 2. a fully saturated one?

Reply:
Not in this PR... that would have to be in the follow-up PR.  And that test case has definitely already been written.

This `defaults_test.go` isn't testing a particular struct, it's testing the behavior of `FillDefaults()` itself.